.github/workflows/snyk.yml

```diff
@@ -0,0 +1,22 @@
+name: Snyk
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  validate:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+    - uses: actions/checkout@master
```

Comment:
Is this using the [v2 beta](https://github.com/actions/checkout) of the checkout action? Should we prefer v1 until v2 is out of beta?